app/assets/stylesheets/mixins/_forum_post.scss

```diff
@@ -0,0 +1,38 @@
+@mixin forum_post {
+  .post {
+    // In case the previous post was too short.
+    clear: both;
+
+    // Spacing between posts.
+    margin-bottom: 1em;
+
+    .user {
+      margin-right: 1em;
+      text-align: right;
+      width: 7em;
+      float: left;
+    }
+
+    .contents {
+      padding: 20px;
+      line-height: 1;
+      border: 1px solid #ddd;
+      border-radius: 4px;
+      box-shadow: 0 1px 1px rgba(0, 0, 0, 0.075);
+
+      // Block formatting context
+      overflow: hidden;
+
+      // Post title
+      .title {
+        margin: -0.4em 0 0.75em 0;
```

Comment:
Properties should be ordered font-size, margin<br>Shorthand form for property `margin` should be written more concisely as `-0.4em 0 0.75em` instead of `-0.4em 0 0.75em 0`